pypeit/reduce.py

```diff
@@ -165,19 +166,19 @@ def _chk_objs(self, items):
 
     def find_objects(self, image, ivar, std=False, ir_redux=False, std_trace=None, maskslits=None,
                           show_peaks=False, show_fits=False, show_trace=False, show=False,
-                     sci_files=None):
+                     manual_extract_dict=None):
 
         sobjs_obj_init, nobj_init, skymask_pos = \
             self.find_objects_pypeline(image, ivar, std=std, std_trace=std_trace, maskslits=maskslits,
                                    show_peaks = show_peaks, show_fits = show_fits, show_trace = show_trace,
-                                       sci_files=sci_files)
+                                       manual_extract_dict=manual_extract_dict)
 
         # For nobj we take only the positive objects
         if ir_redux:
             sobjs_obj_init_neg, nobj_init_neg, skymask_neg = \
                 self.find_objects_pypeline(-image, ivar, std=std, std_trace=std_trace, maskslits=maskslits,
-                show_peaks = show_peaks, show_fits = show_fits, show_trace = show_trace,
-                                           sci_files=sci_files)
+                show_peaks = show_peaks, show_fits = show_fits, show_trace = show_trace)
+                                           #manual_extract_dict=manual_extract_dict)
```

Comment:
Please add the ability in the pypeit file to attach a sign to the manual aperture. In this way you can accommodate the negative objects as well. The positive objects have sign=1, the negative objects have sign = -1. It should be probably be optional information, i.e. the parsing should work if the sign is not there and simply assume positive 1 (for non-near-IR reductions).

Reply:
done;  the negative sign is put on the `det` value.

Reply:
Okay but this sign needs to make it into the sign attribute of the SpecObjs. Did you add that little big of code to objfind?

Reply:
Is it enough to have the append_neg?

Reply:
i.e.

```
sobjs_obj_init.append_neg(sobjs_obj_init_neg)
```